src/components/Pagination/Pagination.tsx

```diff
@@ -108,9 +116,36 @@ export class Pagination extends TsxComponent<Props> {
     this.$emit('page-change', this.selectedPage);
   }
 
+  private keyHandler(e: KeyboardEvent) {
+    const key = e.code;
+    if (key === 'Home') {
+      this.navigateToFirst();
+    } else if (key === 'End') {
+      this.navigateToLast();
+    } else if (key === 'ArrowLeft' || key === 'ArrowUp' || key === 'PageUp') {
+      this.navigateBack();
+    } else if (key === 'ArrowRight' || key === 'ArrowDown' || key === 'PageDown') {
+      this.navigateForward();
+    }
+  }
+
+  public mounted() {
```

Comment:
Have you considered doing it in the render method?

```
  public render() {
    return (
      <div
        class='fd-pagination'
        id='fd-pagination'
        on-keydown={this.keyHandler}
      >
…
```
Then the id attribute can go.

Reply:
Yes, on-keydown does the job of registering event listener. But how about deregistering when the component is destroyed. Do you have idea of how i can refer to the component in destroyed() without using id